src/utils/validateNotes.ts

```diff
@@ -0,0 +1,14 @@
+import { t } from '@i18n'
+
+export const noFirstSymbolsPattern = /^[!"#$%&'()*+,./:;<=>?@[\\\]^_`{|}~§±№-]/
+
+export const validateNotes = (
+  note: string,
+  setError: (arg: string) => void,
+): void => {
+  noFirstSymbolsPattern.test(note)
+    ? setError(t('error.noFirstSymbols'))
+    : setError('')
+
+  if (note.length > 600) setError(t('error.tooLongNote'))
```

Comment:
In your case it could return `{isValid: boolean, error?: string}`
Like so, you can do checks `const validation = isNotesValid(notes); if (!validation.isValid) { setError(validation.error) }`